zcash_primitives/src/transaction/components.rs

```diff
@@ -41,11 +41,7 @@ pub struct OutPoint {
 
 impl std::fmt::Debug for OutPoint {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> Result<(), std::fmt::Error> {
-        write!(
-            f,
-            "OutPoint ({}, {})",
-            super::TxId(self.hash), self.n,
-        )
+        write!(f, "OutPoint ({}, {})", super::TxId(self.hash), self.n,)
```

Comment:
did you mean to include this? (if so, awesome, but figured I'd mention it in case my log changes snuck in)